src/stmt.rs

```diff
@@ -12,6 +12,7 @@ use crate::utils::semicolon_for_stmt;
 
 pub(crate) struct Stmt<'a> {
     inner: &'a ast::Stmt,
+    is_first: bool,
```

Comment:
Do we need to add is_first to get this to work?

Reply:
In my current implementation, yes. I'm not sure if there's a better way to do this, though. Open to suggestions.